.travis.yml

```diff
@@ -1,10 +0,0 @@
-os:
-  - linux
-language: nix
-nix: 2.3.6
```

Comment:
I remember there was a reason to pin the version 2.3.6. Not sure if still necessary but would be a problem to put it fixed just in case?

Reply:
yep, that's pinned now by using `install_url: https://releases.nixos.org/nix/nix-2.3.6/install`